Automattic/wp-calypso
# Upwork Support: Update plan checks

I'm having a bit of trouble with this one.


#### Changes proposed in this Pull Request

* Updates plan checks with Pro plan.

#### Testing instructions



I'm not sure how to functionally test this because Happychat is only available in Production and the fallback is email support.

Hacky testing:
* Add a `console.log(supportVariation);` [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/inline-help/inline-help-contact-view.jsx#L16).
* Apply D80779-code to your sandbox and sandbox public-api.
* Create a new user and site and give it a personal or premium plan.
* Select a mag16 language other than English, like German
* Click on the help icon in the bottom right of the screen.
* Check console for `SUPPORT_UPWORK_TICKET` as the support variation.

Related to p4TIVU-a66-p2
